setup/files/supervisord.conf

```diff
@@ -5,13 +5,13 @@ pidfile=/opt/redash/supervisord/supervisord.pid
 directory=/opt/redash/current
 
 [inet_http_server]
-port = 127.0.0.1:9001
+port = 0.0.0.0:9001
```

Comment:
Considering that you're using a separate file for docker anyway, better to revert the changes to this file.